crates/rpc-server/src/registry.rs

```diff
@@ -178,6 +178,7 @@ pub struct RegistryArgs<T> {
     pub server_config: RPCServerConfig,
     pub chain_config: ChainConfig,
     pub consensus_config: ConsensusConfig,
+    pub gasless_tx_support_config: Option<GaslessTxSupportConfig>,
```

Comment:
Passing this down is getting so cumbersome that I am missing DI frameworks😂.

Why don't we just pass the whole config?